### DISCUSSION TITLE
Battle Eye

### DISCUSSION THREAD

**Discussion by U70U** [UltimateMenu/UltimateMenu#68] [General]
what can we do ?
battle Eye.
![image](https://github.com/user-attachments/assets/908facfa-54ad-4117-8c2a-7722454fa532)

**Comment by L7NEG** [comment]
> what can we do ? battle Eye. ![image](https://private-user-images.githubusercontent.com/176322292/373970293-908facfa-54ad-4117-8c2a-7722454fa532.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyNDEwNTgsIm5iZiI6MTcyODI0MDc1OCwicGF0aCI6Ii8xNzYzMjIyOTIvMzczOTcwMjkzLTkwOGZhY2ZhLTU0YWQtNDExNy04YzJhLTc3MjI0NTRmYTUzMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwNlQxODUyMzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05Y2JiMzE0MWUyZmExNTE3NGQ0YWRkN2MzOGYxNDE5YmI1ODJjNWYyNmI5ODZkNGMyYmIxYTIzZjI3ZWYyZmIxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gGpaIxCFolkp5FEwj4fu0k_lhqPwyv28ASUmzUYNYF8)

follow this tutorial https://github.com/YimMenu/YimMenu/issues/3690#issuecomment-2359702034

**Comment by U70U** [comment]
Thank you body
Free Palestine ... ❤️

**Comment by U70U** [comment]
there is we havnt Full document for install this mod?

**Comment by U70U** [comment]
can i trust it?
---removed--
How can i work with xenos injector?
what stuff i need to install your Lua Mod Menu 
is this enough ?

![image](https://github.com/user-attachments/assets/599e3b08-9edb-4ded-a700-a06aac79a8d7)
![image](https://github.com/user-attachments/assets/1c42e27d-536d-4651-aa7e-12af7d941b5c)

**Comment by U70U** [comment]
which version of Xenos , Script Hook , Lua plugin you suggest ?

**Comment by L7NEG** [comment]
> can i trust it? --removed-- How can i work with xenos injector? what stuff i need to install your Lua Mod Menu is this enough ?
> 
> ![image](https://private-user-images.githubusercontent.com/176322292/373974007-599e3b08-9edb-4ded-a700-a06aac79a8d7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyNDMwMjUsIm5iZiI6MTcyODI0MjcyNSwicGF0aCI6Ii8xNzYzMjIyOTIvMzczOTc0MDA3LTU5OWUzYjA4LTllZGItNGRlZC1hNzAwLWEwNmFhYzc5YThkNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwNlQxOTI1MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYmUxNzY1ZjdkNDY0OTIxZDdlMzlhMjVkODllZGJhYWRlMTFmYmE0MmQ4NDJhMzZiY2YzOTZmMGYwNjVjOTNkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.fmJqAR_3W38pY4dmrxlrtdRkkdCRrjVBjbowA3RBs1k) ![image](https://private-user-images.githubusercontent.com/176322292/373974016-1c42e27d-536d-4651-aa7e-12af7d941b5c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyNDMwMjUsIm5iZiI6MTcyODI0MjcyNSwicGF0aCI6Ii8xNzYzMjIyOTIvMzczOTc0MDE2LTFjNDJlMjdkLTUzNmQtNDY1MS1hYTdlLTEyYWY3ZDk0MWI1Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwNlQxOTI1MjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZTRkZTE5MDJhNDM5ZmQ4ZTgwN2FlYjQzMjcwMzJhMjVjMzY5NjgyZDA0MDExMWM0MzEzOTNjNjA2MDc4ODI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.WXinebBwlXCpiY_ObeuqkWQAknBtUFeS1fi5G0V9Elw)

no the xenos one you sent is a trojan the real one is here https://github.com/DarthTon/Xenos

**Comment by L7NEG** [comment]
+ ultiamte menu is only for YimMenu not for stand or idk if there is a plugin that makes you able to run yimmenu scripts into stand menu

**Comment by U70U** [comment]
then how other people install yimmenu?
i want to learn 
need document or source 
eductional use ( and also need in game money night club loop :| )

**Comment by L7NEG** [comment]
> then how other people install yimmenu? i want to learn need document or source eductional use ( and also need in game money night club loop :| )

just searh on youtube here https://www.youtube.com/watch?v=6vj1H_PPhcQ

for yimmenu dll here https://github.com/Mr-X-GTA/YimMenu

**Comment by U70U** [comment]
thank you akhi

**Comment by L7NEG** [comment]
> thank you akhi

you are welcome bro <3

**Comment by U70U** [comment]
i got this error in injecting 
![image](https://github.com/user-attachments/assets/8e81fb11-c274-4c29-8aad-ed3cd77b770e)

**Comment by L7NEG** [comment]
> i got this error in injecting ![image](https://private-user-images.githubusercontent.com/176322292/373976505-8e81fb11-c274-4c29-8aad-ed3cd77b770e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyNDUyMzQsIm5iZiI6MTcyODI0NDkzNCwicGF0aCI6Ii8xNzYzMjIyOTIvMzczOTc2NTA1LThlODFmYjExLWMyNzQtNGMyOS04YWFkLWVkM2NkNzdiNzcwZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwNlQyMDAyMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MTk5NDVhMDRkNzVkMWZhMjkzZmNmNWZlMjg0YzdiYmNlY2QxNzYzYzU5MzA2YTAzOTFiOTFkZjJjZWU0ZjY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.0ui_KqJWLGa-YyawtSWOJKi6CrHYYaNqLacqyacXAec)

just load your gta into main screen and dont enter online or offline then inject the yimmenu.dll file and done

**Comment by L7NEG** [comment]
make sure you run xenos injector as admin

**Comment by U70U** [comment]
i saw in that video he doing this in game then i do that 
thanks for your new notice

**Comment by L7NEG** [comment]
> i saw in that video he doing this in game then i do that thanks for your new notice

you are welcome bro enjoy <3

**Comment by U70U** [comment]
Im so sorry but..
idk why i do every thing 
![image](https://github.com/user-attachments/assets/5591cf5c-ea3b-4f42-a2c5-7c440376b487)

**Comment by U70U** [comment]
![image](https://github.com/user-attachments/assets/3cf5b041-8c33-48ef-b718-ca83eb3494da)
Battle Eye is On

**Comment by U70U** [comment]
i have no chance ...
thank you bro. i wait for your answer

**Comment by L7NEG** [comment]
> ![image](https://private-user-images.githubusercontent.com/176322292/373977591-3cf5b041-8c33-48ef-b718-ca83eb3494da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyNDg4MjksIm5iZiI6MTcyODI0ODUyOSwicGF0aCI6Ii8xNzYzMjIyOTIvMzczOTc3NTkxLTNjZjViMDQxLThjMzMtNDhlZi1iNzE4LWNhODNlYjM0OTRkYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwNlQyMTAyMDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNjIzZmQ5OTk0NTZiOGYwM2M1ODg2NjNmODU4NmNmMjIyNDc2YjQ4MzFmNjc2ZGYzZjY4ODc4ZTc5MjJiOTRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.OOdwmSD2g8eDAMv0Il5ebK-cpZPgHvuxoOqWcdE2Mf0) Battle Eye is On

just toggle it off and add this commandline.txt into your game folder and done 
[commandline.txt](https://github.com/user-attachments/files/17270890/commandline.txt)

**Comment by aftabtai5352** [comment]
Can I still use Yim mod menu in GTA online, I will use it in invite only session and want money only nothing else.
ik will not earn more than 2.5 mil simultaneously.

is their any safe/secure way to inejct yim or any other mod menu in gta online only????
Please guide me if you have plz

**Comment by L7NEG** [comment]
> Can I still use Yim mod menu in GTA online, I will use it in invite only session and want money only nothing else. ik will not earn more than 2.5 mil simultaneously.
> 
> is their any safe/secure way to inejct yim or any other mod menu in gta online only???? Please guide me if you have plz

Yes you can use it with latest update of fsl file you can download the v1.3 either from unknowncheats.me or this direct file link from stand discord server https://third-party.files-that-make-your-head-explode.com/fsl/1.3/version.dll

**Comment by aftabtai5352** [comment]
> > Can I still use Yim mod menu in GTA online, I will use it in invite only session and want money only nothing else. ik will not earn more than 2.5 mil simultaneously.
> > is their any safe/secure way to inejct yim or any other mod menu in gta online only???? Please guide me if you have plz
> 
> Yes you can use it with latest update of fsl file you can download the v1.3 either from unknowncheats.me or this direct file link from stand discord server https://third-party.files-that-make-your-head-explode.com/fsl/1.3/version.dll


Can you please provide full tutorial, I am flavourglasted , I tried my best to use mod after battleye in online but i am unable to do that. I banned my 2 account only 1 account is left i want to use mod safe and secure. I will use it only in invite only session please guide or give me reference where i found legit tutorial to use mod menu in gta online v1.69 after battleye update.

Thanks for your convienence

**Comment by aftabtai5352** [comment]
When I turn off battleye, and try to enter in online (Ex: game->lobby->Story mode->settings ->online->invite only session)
It shows error "Battleye is required to play gta online" 

what should i do?

**Comment by L7NEG** [comment]
> When I turn off battleye, and try to enter in online (Ex: game->lobby->Story mode->settings ->online->invite only session) It shows error "Battleye is required to play gta online"
> 
> what should i do?

Just use the fsl file above put it as version.dll in the game folder 📂 then just use the yimmenu in the main game screen and make sure BE is off

**Comment by aftabtai5352** [comment]
> > When I turn off battleye, and try to enter in online (Ex: game->lobby->Story mode->settings ->online->invite only session) It shows error "Battleye is required to play gta online"
> > what should i do?
> 
> Just use the fsl file above put it as version.dll in the game folder 📂 then just use the yimmenu in the main game screen and make sure BE is off

Could you please provide me yim menu for gta build 1.69 3351?
This is the last step

**Comment by L7NEG** [comment]
> > > When I turn off battleye, and try to enter in online (Ex: game->lobby->Story mode->settings ->online->invite only session) It shows error "Battleye is required to play gta online"
> > > what should i do?
> > 
> > 
> > Just use the fsl file above put it as version.dll in the game folder 📂 then just use the yimmenu in the main game screen and make sure BE is off
> 
> Could you please provide me yim menu for gta build 1.69 3351? This is the last step

sure here bro https://github.com/Mr-X-GTA/YimMenu

**Comment by L7NEG** [comment]
> > > When I turn off battleye, and try to enter in online (Ex: game->lobby->Story mode->settings ->online->invite only session) It shows error "Battleye is required to play gta online"
> > > what should i do?
> > 
> > 
> > Just use the fsl file above put it as version.dll in the game folder 📂 then just use the yimmenu in the main game screen and make sure BE is off
> 
> Could you please provide me yim menu for gta build 1.69 3351? This is the last step

and for how to use here https://github.com/Mr-X-GTA/YimMenu/issues/26#issuecomment-2435924965

**Comment by L7NEG** [comment]
and for fsl v1.3 just get it from unknowncheats
